server.js

```diff
@@ -7283,6 +7283,48 @@ camp.route(/^\/nsp\/npm\/(?:@([^/]+)?\/)?([^/]+)?(?:\/([^/]+)?)?\.(svg|png|gif|j
   }
 }));
 
+// Redmine plugin rating.
+camp.route(/^\/redmine\/plugins\/(.*)\/(rating|stars)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could you make this `/plugin` instead of `/plugins`, for consistency with our other plugin badges?

Other than that, looks good to go! Sorry I missed that the first time through.

Reply:
@paulmelnikow for consistency, should I   also change uri format like for Chrome web store integration?
Before /redmine/plugins/Plugin-name/rating.svg
After /redmine/plugin/rating/Plugin-name.svg

Reply:
Yes, that would be perfect. Thanks!